huaweicloud/services/evs/resource_huaweicloud_evs_snapshot.go

```diff
@@ -65,6 +65,7 @@ func ResourceEvsSnapshotV2() *schema.Resource {
 			"description": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
```

Comment:
if the description can be left blank, need to remove "Computed".

Reply:
done.